src/asset-catalog/asset-catalog-view.js

```diff
@@ -45,7 +48,7 @@ class AssetCatalogView extends PolymerElement {
 
       <iron-pages selected="[[routeData.usecase]]" attr-for-selected="name" role="main" fallback-selection="search">
         <asset-search-view language="[[language]]" name="search" route="{{tail}}"></asset-search-view>
-        <asset-details-view language="[[language]]" name="view" route="{{tail}}"></asset-details-view>
+        <asset-details-view language="[[language]]" name="view" item="[[item]]"></asset-details-view>
```

Comment:
this property "item" caused confusion in the selectedNode, clickedNode, pop-up node and it should be taken into account in the refactor so that it's clear what data we are passing from top to bottom